Formula/v8.rb

```diff
@@ -78,7 +79,7 @@ def install
     # Build gn from source and add it to the PATH
     (buildpath/"gn").install resource("gn")
     cd "gn" do
-      system "python", "build/gen.py"
```

Comment:
should this be better with `#{Formula["python@3.9"].opt_bin}/python3`?

Reply:
`system "python3"` is fine. This will always be the `python3` dependency declared in the formula given our audit that prevents mixed Python dependencies.